packages/nextjs/scaffold.config.ts

```diff
@@ -0,0 +1,13 @@
+import * as chain from "wagmi/chains";
+
+type TScaffoldConfig = {
+  targetNetwork: chain.Chain;
+  pollingInterval: number;
+};
+
+const ScaffoldConfig: TScaffoldConfig = {
```

Comment:
I think we should settle on a styleguide and apply it on the whole project (I like [google's styleguide](https://google.github.io/styleguide/tsguide.html) as a good starting point).

That means here we'd follow default naming convention and make the const `lowerCamelCase`.

As for the type, but I think the convention should be to not prefix or suffix it and use regular `UpperCamelCase`, since typescript itself already contains the info about what kind of type it is (https://google.github.io/styleguide/tsguide.html#naming-style). 
Prefixes should, in my opinion, be reserved for generic types, to mark a clear distinction between defined types and generics.

@rin-st @carletex any thoughts?

Reply:
I agree 👍

Reply:
I'm up for a having a styleguide.

Besides applying it here, we should create an issue an for it, discuss what guidelines we want to follow..... and make a PR refactoring all the names / types / etc.

Reply:
Opened an issue about it :+1: